packages/rest/test/routes.test.ts

```diff
@@ -241,20 +241,10 @@ describe("rest: routes", () => {
       @get
       @route(":action")
       op colonRoute(): {};
-
-      #suppress "deprecated"
-      @get
-      @autoRoute
-      @segment("actionTwo")
-      @segmentSeparator(":")
```

Comment:
Why not replace the decorators for these, do we already have similar tests for actionSeparator?

Reply:
Good point. I updated this one to show how to do this with an action and actionSegment.